compiler/crates/relay-config/src/typegen_config.rs

```diff
@@ -129,6 +148,10 @@ pub struct TypegenConfig {
     /// A map from GraphQL error name to import path, example:
     /// {"name:: "MyErrorName", "path": "../src/MyError"}
     pub custom_error_type: Option<CustomTypeImport>,
+
+    // Indicates the type to import and use as the context for live resolvers.
```

Comment:
If you use `///` here instead of `//` it will get picked up as part of the description in the json schema we generate. Can use that here and add similar comments to the `ResolverContextTypeInput` type and its variants?